themes/digital.gov/src/scss/new/_topics-button.scss

```diff
@@ -0,0 +1,91 @@
+@use "uswds-core" as *;
+
+.topics-button {
+  margin-bottom: units("105");
+
+  &__link {
+    @include u-font("sans", "sm");
+    @include u-text("medium");
+    @include u-text("no-underline");
+    @include u-bg("white");
+    @include u-text("primary-vivid");
+    @include u-border("1px", "primary-vivid", "solid");
+    @include u-radius("sm");
+    display: block;
+    overflow: auto;
+    padding: units(1);
+    transition: background-color 0.5s, color 0.5s;
+    white-space: nowrap;
+
+    @include at-media("tablet") {
+      padding-left: units(1);
+    }
+  }
+
+  &__arrow {
+    @include u-float("right");
+    @include u-text("gray-1");
+    display: none;
+    margin-right: 10px;
+    transition: margin 0.4s;
+
+    @include at-media("tablet-lg") {
+      display: block;
+    }
+  }
```

Comment:
Changes to the arrow are causing this small regression on desktop.

| [Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-topics-button-component/topics/) | [Live](https://digital.gov/topics/) |
|--------|--------|

![image](https://github.com/GSA/digitalgov.gov/assets/3385219/0732b486-7fe4-430d-9755-6757b7e7dcd1)